account/manager.go

```diff
@@ -1786,7 +1837,7 @@ coinSelection:
 	for {
 		utxos, err := m.cfg.Wallet.ListUnspent(ctx, 1, math.MaxInt32)
 		if err != nil {
-			return nil, nil, nil, err
+			return nil, nil, err
```

Comment:
Can't we just cut this out all together? Since we can reply in the wallet to just do `FundPsbt` and handle all the UTXO selection aspects?